### ISSUE TITLE
Explorer NavBar/CommandBar edit

### ISSUE THREAD

**Issue by Undisputed00x** [Maplespe/ExplorerBlurMica#135]
Hi there although not an effects modification request, i would like to know if its possible to remove both the Navigation Bar + Command Bar of explorer so there would be a minimal but beautiful view of the explorer translucency.
Unfortunately i don't know WinUI/Xaml for now to help but i think that this feature wouldn't be difficult for you to implement. 😄

Below my try i did with UWPSpy 😅
<details><summary>Details</summary>
<p>

![Screenshot 2024-04-06 235606](https://github.com/Maplespe/ExplorerBlurMica/assets/80290008/04c7b4f2-3acc-4af5-b02a-73f542a50d29)

![Screenshot 2024-04-06 235025](https://github.com/Maplespe/ExplorerBlurMica/assets/80290008/35f38d7e-c2a5-4d49-9ce3-f74682063ddd)

</p>
</details>

**Comment by Maplespe** [comment]
You can use StartAllback to restore win7 style title bar and use third party themes.

**Comment by Undisputed00x** [comment]
I have done that even without using [StartAllBack](https://www.startallback.com/)/[EP](https://github.com/valinet/ExplorerPatcher), just by modifying msstyles/shellstyle .
<details><summary>Details</summary>
<p>
![Screenshot 2024-04-07 140846](https://github.com/Maplespe/ExplorerBlurMica/assets/80290008/5c8dda95-f525-4d4f-b1d6-d5bbc2f0df75)
</p>
</details> 
The reason for this request is to keep win11 explorer tabs without WinUI Navbar/Commandbar, StartAllBack can do that only in win7/10 explorer but I dont like StartAllBack because overrides and customizes theme elements like the context menu and EP cant do neither of these things, these projects barely modify XAML elements and EP dev currently fixes bugs and doesnt intend to add features.

**Comment by Maplespe** [comment]
Direct hiding will result in empty content.

**Comment by Undisputed00x** [comment]
> Direct hiding will result in empty content.

Yes but removing wouldn't be able to adjust the explorerframe area ?

**Comment by Undisputed00x** [comment]
Now possible with https://windhawk.net/mods/windows-11-file-explorer-styler

Issue closes.
